### ISSUE TITLE
Show balance in dashboard

### ISSUE THREAD

**Issue by wirednkod** [delegit-xyz/dashboard#182]
The balance that is avalaible, froze etc etc should be displayed in the dashboard (feature request)

**Comment by Tbaut** [comment]
I disagree. I think the only place it's needed if for when you delegate, and this is this issue https://github.com/delegit-xyz/dashboard/issues/28

**Comment by wirednkod** [comment]
A damn.. did not see #28 . Thnx for that!
